src/parser/jobs/blu/modules/Overheal.tsx

```diff
@@ -0,0 +1,77 @@
+import {Trans} from '@lingui/react'
+import ACTIONS from 'data/ACTIONS'
+import {Event} from 'event'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Actors} from 'parser/core/modules/Actors'
+import {Overheal, SuggestedColors} from 'parser/core/modules/Overheal'
+import React from 'react'
+
+export class BLUOverheal extends Overheal {
+
+	@dependency private actors!: Actors
```

Comment:
Unused dependency, can get rid of this.

Reply:
Patched in 6c55c596f9cf850dc7756243e0fe541b939e343f